docker-compose.yaml

```diff
@@ -24,6 +24,20 @@ services:
       retries: 3
       start_period: 10s
 
+  pgadmin:
+    container_name: pgadmin
+    image: dpage/pgadmin4:latest
+    environment:
+      PGADMIN_DEFAULT_EMAIL: clair@clair.com
+      PGADMIN_DEFAULT_PASSWORD: clair
+      PGADMIN_SERVER_JSON_FILE: /pgadmin4/config/servers.json
+    ports:
+      - "80:80"
```

Comment:
Can we move this to a less common port? Taking port 80 seems a bit brazen and requires sudo.

Reply:
Yeah, I also think so. What do you think would be a good port number, @ldelossa ? I was thinking maybe 8081 since we already use 8080 and 8082.